pkg/manifests/manifests.go

```diff
@@ -965,6 +968,26 @@ func (f *Factory) generateHtpasswdSecret(s *v1.Secret, password string) {
 	s.Namespace = f.namespace
 }
 
+func (f *Factory) ThanosRulerQueryConfigSecret() (*v1.Secret, error) {
```

Comment:
let's add this to `manifests_test.go`

Reply:
You mean to add to the TestUnconfiguredManifests test cases?

Reply:
Added to the tests.